cmd/stack_package.go

```diff
@@ -88,9 +91,43 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 			Info.Log("Running appsody stack package")
 			Info.Log("******************************************")
 
-			stackPath := rootConfig.ProjectDir
+			projectPath := rootConfig.ProjectDir
+
+			// make a copy of the folder to apply template to
+			err := copyDir(projectPath, projectPath+"packagecopy")
+			if err != nil {
+				os.RemoveAll(projectPath + "packagecopy")
+				return errors.Errorf("Error trying to copy directory: %v", err)
+			}
+
+			// remove copied folder locally, no matter the output
+			defer os.RemoveAll(projectPath + "packagecopy")
+
+			// sets stack path to be the copied folder
+			stackPath := projectPath + "packagecopy"
 			Debug.Log("stackPath is: ", stackPath)
 
+			// get the stack name from the stack path (removes copy suffix)
+			stackID := filepath.Base(strings.Replace(stackPath, "packagecopy", "", 1))
```

Comment:
You have the `projectPath` variable above, use that here instead of the strings.Replace()